helmfile.d/values/networkpolicies/common/prometheus.yaml.gotmpl

```
@@ -33,6 +33,18 @@ policies:
         - rule: ingress-rule-apiserver
           ports:
             - tcp: 9090
+        {{- if and .Values.prometheus.internalAccess.enabled .Values.prometheus.internalAccess.namespaces }}
+        {{- range $namespace := .Values.prometheus.internalAccess.namespaces }}
+        - name: {{ $namespace }}-ingress-rule-internal-prometheus
+          peers:
+            - namespaceSelectorLabels:
+                kubernetes.io/metadata.name: {{ $namespace }}
+              podSelectorLabels:
+                elastisys.io/prometheus-access: allow
+          ports:
+            - tcp: 9090
+        {{- end }}
+        {{- end }}
```

```suggestion
        {{- if and .Values.prometheus.internalAccess.enabled .Values.prometheus.internalAccess.namespaces }}
        - name: ingress-rule-internal-prometheus
          peers:
            {{- range $namespace := .Values.prometheus.internalAccess.namespaces }}
            - namespaceSelectorLabels:
                kubernetes.io/metadata.name: {{ $namespace }}
              podSelectorLabels:
                elastisys.io/prometheus-access: allow
            {{- end }}
          ports:
            - tcp: 9090
        {{- end }}
```
To reduce the size of the manifest.

---

So something like `.networkPolicies.monitoring.prometheus.internalAccess`?

---

Maybe, I guess we can leave it open and see what other people think about it.

---

I agree that this probably fits better under networkPolicies

---

I agree on that we should move the feature flag to under `.networkPolicies` to be consistent.